easy/solution_9.sql

```diff
@@ -0,0 +1,8 @@
+-- 
+-- Find the youngest athlete in the dataset.
+SELECT 
+    name, 
+    age 
+FROM public.olympics_history
+ORDER BY age ASC
+LIMIT 1;
```

Comment:
```suggestion
LIMIT 1;
```
This skips other athletes of the same age. Need to rewrite.

Reply:
Use `RANK`.